src/components/Navigation/Navigation.spec.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react'
+import Navigation from './Navigation'
+import { BrowserRouter } from 'react-router-dom'
+
+describe('Navigation', () => {
+  it('renders links to "MyCloset" and "New"', () => {
+    render(
+      <BrowserRouter>
+        <Navigation />
+      </BrowserRouter>
```

Comment:
The official React Router documentation uses `MemoryRouter` for testing purposes.
Since I am not highly familiar with React Router yet, you could check [this site ](https://reactrouter.com/web/guides/testing) for more information and please let me know if I am wrong :)